mix.exs

```diff
@@ -30,8 +30,8 @@ defmodule PaperTrail.Mixfile do
 
   defp deps do
     [
-      {:ecto, ">= 3.6.0"},
-      {:ecto_sql, ">= 3.6.0"},
+      {:ecto, ">= 3.8.0"},
```

Comment:
the other discussion was about not making it a breaking change, but here we are saying this lib now requires ecto 3.8, which would break any project using ecto < 3.8.

Can't we keep it as >= 3.6.0 since we made the change to support both return types?